Idno/start.php

```diff
@@ -94,9 +94,15 @@ function &loader()
 
 // Bonita is being used for templating
     $known_loader->registerNamespace('Bonita', dirname(dirname(__FILE__)) . '/external/bonita/includes');
+//PSR Log Interface    
+    include dirname(dirname(__FILE__)) . '/external/Psr/Log/LoggerInterface.php';
+    include dirname(dirname(__FILE__)) . '/external/Psr/Log/AbstractLogger.php';
+    include dirname(dirname(__FILE__)) . '/external/Psr/Log/LogLevel.php';
+
```

Comment:
@kylewm If you see my previous commit i used the registerNamespace function but it did not work idk why , please do try and tell me if it worked for you